### DISCUSSION TITLE
`.env` support

### DISCUSSION THREAD

**Discussion by amitu** [fastn-stack/fastn#1384] [Ideas & RFCs]
It would be great if `fastn` looked for a file `.env` and used environment variables from that file, so people do not have to use other ways to export environment variables. 

This file should not be checked-in. So it poses a risk that some people will accidentally check in this file and leak secrets. 

We can not keep secrets in `FASTN.ftd` file, and environment variables are better way to manage secrets. To try to save users from making mistakes we can do a git check, if .env is committed or added to index we do not use the file and report a warning, and only accept a a checked in .env if  `FASTN_DANGER_ACCEPT_CHECKED_IN_ENV` environment variable is set.

**Comment by siddhantk232** [comment]
#1424
